pkgs/by-name/do/dosbox-staging/package.nix

```diff
@@ -57,6 +58,39 @@ stdenv.mkDerivation (finalAttrs: {
       hash = "sha256-WtTVSWWSlfXrdPVsnlDe4P5K/Fnj4QsOzx3Wo/Kusmg=";
       includes = [ "src/gui/meson.build" ];
     })
+  ]
+  # Pagesize detection via syscall; remove when next stable version arrives
+  ++ [
```

Comment:
```suggestion
  # Pagesize detection via syscall; remove when next stable version arrives
```

Reply:
I put this comment outside the list because all the patches deal with this.